src/http.rs

```diff
@@ -112,7 +112,7 @@ where
         let client = client_builder.build(proxy_connector.clone());
 
         let version = crate::get_version();
-        let user_agent = HeaderValue::from_str(&format!("Mezmo/{version}"))
+        let user_agent = HeaderValue::from_str(&format!("MezmoEdge/{version}"))
```

Comment:
I think we should make this configurable to avoid diverging across repos

```rs
let user_agent = mezmo_env_config!("MEZMO_HTTP_USER_AGENT", "Mezmo")
let user_agent = HeaderValue::from_str(&format!("{user_agent}/{version}"))
```

Then it can be overridden in the helm chart for edge